Assets/!/Scripts/Player/PlayerAudio.cs

```diff
@@ -1,11 +1,21 @@
-using System.Diagnostics;
+using System.Collections.Generic;
 using UnityEngine;
 using utils;
 
 public class PlayerAudio : MonoBehaviour
 {
     public AudioSource audioSource;
-    public Audio[] audioPlayer;
+
+    public Audio[] stepAudio;
+    public Audio[] dieScreamAudio;
+    public Audio[] bloodAudio;
+    public Audio[] crawlAudio;
+    public Audio[] bulletHitAudio;
+    public Audio[] jumpAudio;
+    public Audio[] fallDownAudio;
+    public Audio[] fireK98Audio;
+    public Audio[] k98CycleAudio;
+    public Audio[] smokeAudio; 
```

Comment:
rename to voice lines

Reply:
Should we leave the name for now?

Reply:
I think it is about renaming the Array. The Method will be renamed later.

Reply:
renamed now